jwst/photom/photom.py

```diff
@@ -618,21 +613,19 @@ def photom_io(self, tabdata):
         -------
 
         """
-        # Get the conversion factor from the PHOTMJSR column of the table row
-        conv_factor = tabdata['photmjsr']
+        # Get the scalar conversion factor from the PHOTMJSR column of the table row
+        conversion = tabdata['photmjsr']
 
-        # Store the conversion factors in the meta data
-        log.info('PHOTMJSR value: %g', conv_factor)
+        # Store the conversion factor in the meta data
+        log.info('PHOTMJSR value: %g', conversion)
         if isinstance(self.input, datamodels.MultiSlitModel):
             self.input.slits[self.slitnum].meta.photometry.conversion_megajanskys = \
-                conv_factor
+                conversion
             self.input.slits[self.slitnum].meta.photometry.conversion_microjanskys = \
-                23.50443 * conv_factor
+                23.50443 * conversion
```

Comment:
Would it be better to have numbers such as 23.50443 defined as a constant in the file?

Reply:
Or to use astropy units for the conversion?

Reply:
Astropy units used in latest commit.